### DISCUSSION TITLE
Migrating from an existing django-tenant-schemas application

### DISCUSSION THREAD

**Discussion by jeroenbrouwer** [lorinkoz/django-pgschemas#296] [Q&A]
Hi there, 

The `django-tenant-schemas` package has been unmaintained for a while. This package looks like an interesting replacement. Would it be possible to migrate an existing django application which uses `django-tenant-schemas` to using `django-pgschemas` without much trouble? 

It might be a good idea to put an answer to this question in the docs. The same goes for migrating from `django-tenants`.

**Comment by lorinkoz** [comment]
I think the biggest obstacle for migrating into `pgschemas` would be moving the non-shared models from the `public` schema into a statically defined schema (e.g. `www`, `main`). The rest is more less configuration and should be convertible.

As I don't have the experience myself with a migration like this, I wouldn't be able to provide helpful docs. But if you decided to take the path yourself we could add something to the docs from your experience.
